msal/broker.py

```diff
@@ -9,6 +9,10 @@
 
 
 logger = logging.getLogger(__name__)
+
+from .application import (
+    __version__)
+
```

Comment:
[In Python, the convention](https://peps.python.org/pep-0008/#imports) is to group all imports at the beginning of the file.

And, now that you ended up duplicating the sku magic string, so you should have go ahead and define a const in application.py for this change.

```suggestion

from .application import __version__, SKU


logger = logging.getLogger(__name__)


```